getting-started/_posts/2017-01-13-introduction-to-docker-compose.markdown

```diff
@@ -0,0 +1,141 @@
+---
+layout: page
+title: Introduction to Docker Compose
+category: getting-started
+permalink: /introduction-to-docker-compose
+step: 10
+tags:
+- getting-started
+- compose
+- docker
+excerpt: Learn how Docker Compose allows us to define multi-container environments.
+---
+
+Docker Compose allows you to define and run multiple-container Docker environments. You're able to configure your application services in a Compose file, and then start and stop all services from this configuration file.
+
+Compose uses [YAML](http://yaml.org) (YAML Ain't Markup Language), a human-friendly data serialization standard, for its configuration file format.
+
+## Why Use Docker Compose
+Compose allows you to:
+- Create multiple isolated environments on a single host
```

Comment:
I think you need another newline before this list in order for it to be formatted correctly.

![image](https://cloud.githubusercontent.com/assets/7440805/22002601/250be384-dc02-11e6-8308-a0baca5aab4a.png)